### ISSUE TITLE
How to run Torch on distributed systems?

### ISSUE THREAD

**Issue by SergeyMalashenko** [torch/torch7#919]
Good day!
We have three servers with Titan-X and we want to run some Torch code with different parameters on every server. How to do it?

**Comment by soumith** [comment]
Hi, you can use this package to do distributed learning with torch: https://github.com/twitter/torch-distlearn

**Comment by SergeyMalashenko** [comment]
You are not right. I want to run one script but in different modes on the first, the second and the third server after that I want to collect result data ( mean error, recall ) and do some additional actions.
